### PR TITLE
Hotfix/v1.1.0: Updated the Changelogfile

### PR DESCRIPTION
**Pull request by waji46**
**Title: Updated the Changelog**

    [PS-1]
     - Updated the Changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Added the EIC endpoint
+- Added env condition for infra
+- Added the automation script
+- Updated the Architecture Diagram in the Readme file
+- Updated the Readme file
+
+
+### Fixed
+- Linked List Lambda issue fixed
+- Fixed some issues in the pre-req.sh bash script
```
Comment (faheem-khau9) [inline CHANGELOG.md:22]: Mention the issues as well

**Review by faheem-khau9 — CHANGES_REQUESTED** [review]

**Comment by samanXgrid** [comment]
it should be from develop to main

**Review by yaseenzafar123 — DISMISSED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (waji46) [reply CHANGELOG.md]: Added in 51cb5a12dd43a8abb2932db2fa4de5d270cb1edf

**Review by yaseenzafar123 — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Added the EIC endpoint
+- Added env condition for infra
+- Added the automation script
+- Updated the Architecture Diagram in the Readme file
+- Updated the Readme file
+
+
+### Fixed
+- Linked List Lambda issue fixed
+- Added checks for the packages [AWS cli, pyhton, terraform] in the init.sh bash script.
```
Comment (faheem-khau9) [inline CHANGELOG.md:22]:
```suggestion
- Added checks for the packages installation [AWS CLI, python, terraform] in the init.sh bash script.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+### Changed
+- Added the EIC endpoint
+- Added env condition for infra
+- Added the automation script
+- Updated the Architecture Diagram in the Readme file
+- Updated the Readme file
+
+
+### Fixed
+- Linked List Lambda issue fixed
```
Comment (faheem-khau9) [inline CHANGELOG.md:21]:
```suggestion
- Fixed Linked List Lambda to run from a member account as well
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [1.1.0]  - 2023-07-17
+
+
+### Changed
+- Added the EIC endpoint
```
Comment (faheem-khau9) [inline CHANGELOG.md:13]:
```suggestion
- Integrated EIC endpoint and removed bastion host
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Added env condition for infra
```
Comment (faheem-khau9) [inline CHANGELOG.md:14]:
```suggestion
- Updated the infra-deployment based on environment variable
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Added the automation script
```
Comment (faheem-khau9) [inline CHANGELOG.md:15]:
```suggestion
- Added the automation script to deploy XC3 infrastack
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Updated the Architecture Diagram in the Readme file
```
Comment (faheem-khau9) [inline CHANGELOG.md:16]:
```suggestion
- Updated the Architecture Diagram of XC3 in the Readme file
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Updated the Readme file
```
Comment (faheem-khau9) [inline CHANGELOG.md:17]:
```suggestion
```

**Review by faheem-khau9 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -6,6 +6,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Integrated EIC endpoint and removed bastion host
+- Updated the infra-deployment based on environment variable
+- Added the automation script to deploy XC3 infrastack
+- Updated the Architecture Diagram of XC3 in the Readme file
+
+
+### Fixed
+- Fixed Linked List Lambda to run from a member account as well
+- Added checks for the packages installation [AWS CLI, python, terraform] in the init.sh bash script.
+
```
Comment (faheem-khau9) [inline CHANGELOG.md:22]:
```suggestion
```

**Review by faheem-khau9 — CHANGES_REQUESTED** [review]

**Review by faheem-khau9 — APPROVED** [review]

**Review by yaseenzafar123 — APPROVED** [review]
lgtm

**Review by samanXgrid — APPROVED** [review]

**Pull request merged**